template/.github/workflows/docs.yml

```diff
@@ -42,6 +42,11 @@ jobs:
   docs:
     name: Build documentation
     runs-on: 'ubuntu-22.04'
+    env:
+        # Security note! The secrets are only added to workflows that run on trusted branches (main). If secrets are accessible in workflows that run on untrusted branches they can be extracted, see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#exfiltrating-data-from-a-runner.
```

Comment:
This comment is no longer correct

Reply:
[It never was :open_mouth:](https://en.meming.world/images/en/0/08/Always_has_been.jpg)